src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -1332,14 +1334,14 @@ public void LanguageVersionAdded_Canary()
             // - update the "UpgradeProject" codefixer
```

Comment:
If I understand the fixer correctly, this will have to wait until any of the 7.1 features actually make it to master.
I'll log an item for it in the master issue.

---
In reply to: [106310848](https://github.com/dotnet/roslyn/pull/17894#discussion_r106310848) [](ancestors = 106310848)